src/hl7v2/config.clj

```diff
@@ -2,4 +2,5 @@
 
 (def config
   (delay
-    {:standard-dir "tmp"}))
\ No newline at end of file
+    {:standard-dir "tmp"
+     :lang "en"}))
```

Comment:
i think that `:lang` means language and "en" means English, right?

Reply:
correct!